installer/conf/sudoers

```diff
@@ -29,4 +29,8 @@ omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsof
 
 #run hdinsightmanifestreader.rb
 omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsagent/ruby/bin/ruby /opt/microsoft/omsagent/bin/hdinsightmanifestreader.rb
+
+#run update management commands
+omsagent ALL=(ALL) NOPASSWD: /usr/bin/apt-get * 
```

Comment:
It exposes too many permissions (install, update, upgrade, etc) to the omsagent. 
Can you give a list of commands that will be used by PatchManagement?

BTW, we need to update the release document about this change.
@agup006 @KrisBash